webpack.dev.config.js

```diff
@@ -1,5 +1,10 @@
-var common = require('./webpack.common.config');
+/* global module */
+
+var common    = require('./webpack.common.config');
 var devConfig = common('okta-sign-in.js');
+var plugins   = require('./buildtools/webpack/plugins');
```

Comment:
nit: require statements should be above others (group them together) - there's a lint rule for this, but looks like we haven't enabled it here yet.

Reply:
There are a few common eslint rules we aren't including, so I'll submit a new PR with `.eslintrc` updates soon!